packages/table/src/table.tsx

```diff
@@ -1826,6 +1826,11 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
 
         // change selection to match new focus cell location
         const newSelectionRegions = [Regions.cell(newFocusedCell.row, newFocusedCell.col)];
+        const { selectedRegionTransform } = this.props;
+        if (selectedRegionTransform != null) {
+            // tslint:disable-next-line no-unnecessary-callback-wrapper
```

Comment:
do we have to disable this? if so, leave another //comment explaining why.

Reply:
You are correct -- there is no need to disable this. Shall I add another commit to this PR to remove this line and will you squash it if it gets merged eventually?

Reply:
yes add commits. do not force push on PRs.